ansible-ipi-install/roles/installer/templates/install-config.j2

```diff
@@ -31,6 +31,9 @@ platform:
     apiVIP: {{ apivip }}
     ingressVIP: {{ ingressvip }}
     dnsVIP: {{ dnsvip }}
+{% if prov_bridge is defined %}
+    provisioningBridge: {{ prov_bridge }}
+{ % endif %}
```

Comment:
We should be able to use the default filter https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#defaulting-undefined-variables 

Let's change these lines to:
```
provisioningBridge: {{ provisioning_bridge | default('provisioning') }}
```

Reply:
Why don't we set the default value in a fact to avoid using the default filter every time? Like we are doing here https://github.com/openshift-kni/baremetal-deploy/blob/master/ansible-ipi-install/roles/node-prep/tasks/10_validation.yml#L198-L204